fe/fe-core/src/main/java/com/starrocks/planner/StreamLoadScanNode.java

```diff
@@ -267,27 +267,24 @@ public void finalizeStats(Analyzer analyzer) throws StarRocksException, StarRock
     }
 
     private void assignBackends() throws StarRocksException {
-        backends = Lists.newArrayList();
+        computeNodes = Lists.newArrayList();
         if (enableBatchWrite) {
             for (long backendId : batchWriteBackendIds) {
-                Backend backend = GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo().getBackend(backendId);
-                if (backend == null) {
+                ComputeNode computeNode = GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo().getBackendOrComputeNode(backendId);
```

Comment:
can ComputeNode be selected under shared-nothing mode? If not, shall use `RunMode.isSharedDataMode()` to distinguish the two different modes.

Reply:
Fixed